src/bp/ui-studio/src/web/reducers/core.ts

```diff
@@ -0,0 +1,23 @@
+import { handleActions } from 'redux-actions'
+import { licensingReceived } from '~/actions'
+
+export interface CoreReducer {
+  licensing: any
+}
+
+const defaultState = {
+  licensing: undefined
+}
+
+const reducer = handleActions(
+  {
+    [licensingReceived]: (state, { payload }) => {
+      window.LICENSING = { isPro: payload.isPro }
```

Comment:
Why set it on the window ? You can just access the store to get the value.. Also, you could just store "isPro" directly instead of an object with only this property

Reply:
do modules have access to the store? couldn't find an example

Reply:
I think this is something we should pass as top level props to module

Reply:
I didn't find it at the top level of the window element, not sure if it could be deeper in something else in it..?